docs/README.md

```diff
@@ -18,53 +18,45 @@ Help server: https://discord.gg/XGXydQyKhQ
 |  :warning:   | We do not take responsibility for a possible leak of your passwords, which is why you need to host the bot yourself. |
 
 ## Installation
+
 ```sh
 git clone https://github.com/drawbu/drawbot
+cd drawbot
 
-cd /path
-
-pip install -r requirements.txt
+pip install -e .
 ```
 
 ## Launch
 ```sh
-py run.py
+py drawbot
 ```
 
+*If you are using a Linux distribution, you can use the `make` command to install dependencies and run the bot.*
+
 ## Files documentation
 
-The bot will create **2** json files in the **app** folder:
+The bot will create **2** json files in the **vars** folder:
 
--   config.json
 -   devoirs.json
 -   grades.json
 
-### app/ config.json
+### vars/config.json
 
-This file stocks your private logins and info's to make to bor running:
+This file stocks your private logins and info's to make to bot running.
+You can find a **config.example.json** file in the **vars** folder.
 
-```json5
```

Comment:
Why deleting this?

Reply:
I put a `config.json.example` file people can directly copy and rename. Since the content is within the example file, it isn't much useful in the readme, i believe 

Plus having a example file within the vars folder should avoid people to try creating a config.json at another locater, i hope

Reply:
Yep it will be easier for some, but harder for a majority. People using this bot are in majority beginners, so we need to make this accessible. I'm just gonna adding it back, and we'll have both available

Reply:
Sure